File: src/main/java/cn/zjiali/robot/entity/response/FortuneResponse.java
package cn.zjiali.robot.entity.response;

import lombok.Data;

/**
 * 运势实体
 *
 * @author zJiaLi
 * @since 2020-10-31 22:54
 */
@Data
public class FortuneResponse {

    private int status;

    private String message;

    private DataResponse dataResponse;

    @Data
    public static class DataResponse {

        private Long id;

        /**
         * 运情总结
         */
        private String fortuneSummary;

        /**
         * 幸运星
         */
        private String luckyStar;

        /**
         * 签文
         */
        private String signText;

        /**
         * 解签
         */
        private String unSignText;
    }


}

File: src/main/java/cn/zjiali/robot/model/response/FortuneResponse.kt
package cn.zjiali.robot.model.response

/**
 * 运势实体
 *
 * @author zJiaLi
 * @since 2020-10-31 22:54
 */
open class FortuneResponse {
    var status = 0
    var message: String? = null
    var dataResponse: DataResponse? = null


    class DataResponse {
        var id: Long? = null

        /**
         * 运情总结
         */
        var fortuneSummary: String? = null

        /**
         * 幸运星
         */
        var luckyStar: String? = null

        /**
         * 签文
         */
        var signText: String? = null

        /**
         * 解签
         */
        var unSignText: String? = null

    }
}